sphinx_gallery/gen_rst.py

```diff
@@ -208,6 +208,18 @@ def _sanitize_rst(string):
     string = re.sub(p + r'`([^`]+)`' + e, r'\1\2\3', string)
     # `whatever thing` --> whatever thing
     string = re.sub(p + r'([^`]+)' + e, r'\1\2\3', string)
+
+    # **string** --> string
+    string = re.sub(r'\*\*([^\*]*)\*\*', r'\1', string)
+    # *string* --> string
+    string = re.sub(r'\*([^\*]*)\*', r'\1', string)
+    # `link text <url>`_ --> link text
+    string = re.sub(r'`(.*) <.*>`\_', r'\1', string)
```

Comment:
This will also need adjustments like `[^`<>]+` instead of the `.*` inside the group, similar to below

Reply:
... and links can be anonymous with two trailing underscores, so maybe
```suggestion
    string = re.sub(r'`([^`<>]+) <[^`<>]>`\_\_?', r'\1', string)
```